appengine/javatests/com/google/auth/appengine/MockAppIdentityService.java

```diff
@@ -72,7 +74,11 @@ public void setExpiration(Date expiration) {
 
   @Override
   public SigningResult signForApp(byte[] signBlob) {
-    return null;
+    return signingResult;
+  }
+
+  public void setSignature(byte[] signature) {
+    this.signingResult = new SigningResult("keyName", signature);
```

Comment:
It looks like throughout this library, strings are defined as constants instead of being hardcoded - could you make a constant for this case?